### PR TITLE
Data integrity testing + metabase setup + PSOP v2

### PR DESCRIPTION
**Pull request by thala321**
Data integrity:
- Setup The-graph and Massbit Indexer in `ssh massbit@161.97.86.197` to test quickswap
- Mount Postgres volume to the host so we don't lose data when docker is destroyed

Metabase:
- Add back Postgres Dockerfile (someone must have deleted it by accident)
- Add Metabase Dockerfile + script to automatically setup new account for Dockerfile
  ```
   email: admin@massbit.co
   password: Massbit123
   (^ this account isn't real, no worry about information getting leaked)
  ```

Setup quickswap + pancakeswap tests for PSOP

----
Endpoints:
```
The-graph GraphQL Endpoint: http://staging.massbit.io:8000/subgraphs/name/sameepsi/quickswap/graphql
Massbit GraphQL Endpoint: http://staging.massbit.io:8080/v1/graphql
Massbit Dashboard: http://staging.massbit.io:8088
Metabase: http://staging.massbit.io:3002/browse
Hasura Console: not up yet
```

### CODE REVIEW

**Comment by thala321** [comment]
We got quickswap deploy to the-graph

Screenshot from 2021-09-23 13-10-47

File: store/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+FROM postgres
```
Comment (anhhuy0501) [inline store/Dockerfile:1]: Todo: remove store folder.

**Review by anhhuy0501 — APPROVED** [review]
LGTM

File: store/Dockerfile (hunk 1 above)
Comment (thala321) [reply store/Dockerfile:1]: Thanks. I'll refactor in next PR

**Pull request merged**